src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/FixAllContext.cs

```diff
@@ -163,23 +216,55 @@ public async Task<ImmutableArray<Diagnostic>> GetDocumentDiagnosticsAsync(Docume
             }
 
             var getDiagnosticsTask = State.DiagnosticProvider.GetDocumentDiagnosticsAsync(document, this.CancellationToken);
-            return await GetFilteredDiagnosticsAsync(getDiagnosticsTask, this.DiagnosticIds).ConfigureAwait(false);
+            return await GetFilteredDiagnosticsAsync(getDiagnosticsTask, this.DiagnosticIds, filterSpan: null).ConfigureAwait(false);
         }
 
-        private static async Task<ImmutableArray<Diagnostic>> GetFilteredDiagnosticsAsync(Task<IEnumerable<Diagnostic>> getDiagnosticsTask, ImmutableHashSet<string> diagnosticIds)
+        private static async Task<ImmutableArray<Diagnostic>> GetFilteredDiagnosticsAsync(
+            Task<IEnumerable<Diagnostic>> getDiagnosticsTask,
+            ImmutableHashSet<string> diagnosticIds,
+            TextSpan? filterSpan)
         {
             if (getDiagnosticsTask != null)
             {
                 var diagnostics = await getDiagnosticsTask.ConfigureAwait(false);
                 if (diagnostics != null)
                 {
-                    return diagnostics.Where(d => d != null && diagnosticIds.Contains(d.Id)).ToImmutableArray();
+                    return diagnostics.Where(d => d != null && diagnosticIds.Contains(d.Id)
```

Comment:
```suggestion
                    return diagnostics.WhereAsArray(d => d != null && diagnosticIds.Contains(d.Id)
```

Reply:
`WhereAsArray` is an extension method on `ImmutableArray<T>`, but not available on `IEnumerable<T>`